### ISSUE TITLE
Strange Error

### ISSUE THREAD

**Issue by 0xspringtime** [DillonWall/generate-batch-audio-anki-addon#1]
While testing out add-on, became stuck in "Processing" loop wherein I could not close anki even by killing the process in the terminal and had to restart my computer. Unlike anything I've experienced with anki.

Version ⁨2.1.56 Arch

**Comment by DillonWall** [comment]
I'm sorry that you ran into this issue. That is indeed a very strange error. I'm curious, when you were running the addon, what delay were you using, and did you have the "Dangerously Fast" option checked?

**Comment by 0xspringtime** [comment]
I did not have "Dangerously Fast" checked and had the default delay of 0. I have continued using the add-on and haven't encountered the same issue, the only other one being the add-on crashing when removing multiple sources quickly. Otherwise I'm impressed with the add-on and have used it to implement voicevox support in anki which you can see here:
https://github.com/0xspringtime/voicevox-anki

**Comment by DillonWall** [comment]
Oh awesome! I developed this add-on with the goal of making it as flexible as possible, so I'm glad you were able to use it in such an interesting way. 

As for the error, I have a suspicion of what might be going on. 
However, if the error is pretty rare, then it may be a while before I get around to fixing it because this add-on is mainly a quick fix I made to fulfill a need that I and a few others had to bulk fill audio to their cards. 
On the other hand, I do have a lot of ideas for improving it, so if there is a large enough demand, or if you would like a critical feature implemented, then I wouldn't mind returning to this project.
